src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx

```diff
@@ -9,7 +9,7 @@ Use this document to learn how to review and peer edit docs for you fellow write
 
 ## Peer editing workflow in GitHub [#peer-edit]
 
-If you’re peer editing a doc or have been otherwise assigned to a PR as a reviewer, you have a few choices for how and where to do the work.  The most streamlined and open-source approach is to do the edit using GitHub options, rather than copying the file to Google Docs and editing there. 
+If you’re peer editing a doc or have been otherwise assigned to a PR as a reviewer, you have a few choices for how and where to do the wor. The most streamlined and open-source approach is to do the edit using GitHub options, rather than copying the file to Google Docs and editing there. 
```

Comment:
@austin-schaefer, a "k" is missing from "work."

Reply:
sheesh, nice catch.